aws/resource_aws_cognito_user_group.go

```diff
@@ -0,0 +1,175 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+	"github.com/hashicorp/errwrap"
```

Comment:
Nitpick: I know it's used all over the AWS provider at the moment, but in most places for no particular reason. The library is intended to address the problem of wrapping errors into errors so that we can retain all data about the original error. Here we're just wrapping errors to _print_ them out later. In other words I think we don't need this library unless we use `Contains()` or `ContainsType()` per https://github.com/hashicorp/errwrap#usage